pubspec.yaml

```diff
@@ -27,7 +27,7 @@ dev_dependencies:
   test: '^0.12.18+1'
   stack_trace: '>=0.9.1 <2.0.0'
 environment:
-  sdk: ">=1.9.0 <2.0.0"
+  sdk: ">=2.0.0-dev <3.0.0"
```

Comment:
Isn't this change compatible with `>=1.22.0`

Reply:
nice catch.
They should be. Will undo the version update.

Reply:
Updated to 1.24, since it requires generic type support (even if they are ignored).